<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TAWizard</class>
 <widget class="QWizard" name="TAWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Allocate Travelling Allowance</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Travelling Allowance</string>
   </property>
   <property name="subTitle">
    <string>Welcome. Please enter the travelling allowance details.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Payroll Month</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboMonths"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="lstEmployees">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Roll #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Travelling Allowance Details</string>
   </property>
   <property name="subTitle">
    <string>Please enter the travelling allowance issued to each employee.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tblTA">
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount Issued</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EmployeeID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <property name="title">
    <string>Finish</string>
   </property>
   <property name="subTitle">
    <string>Please click on the save button to save the travelling allowance.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="text">
       <string>Save</string>
      </property>
      <property name="description">
       <string>Click here to save the travelling allowance</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>277</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
